Sources/Extensions/Foundation/Double+Povio.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Double+Povio.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 11/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import Foundation
+
+public extension Double {
+  typealias Meters = Double
+  typealias Miles = Double
+  
+  /// Returns miles in meters
+  var meters: Meters {
+    self * 1609.344
+  }
+  
+  /// Return meters in miles
+  var miles: Miles {
+    self / 1609.344
+  }
+}
```

Comment:
Not sure I like this, we can easily get into a situation where we do the same conversion twice :\ (meters to meters for example)

Should we have proper types wrapping this and then make conversions between those?

Reply:
Thinking further about this if this makes sense at all. I'll think about it.

Reply:
@nejcvivod I've decided to remove it since it's not clear what it does with the current state.